vault/Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN make bootstrap && \
     make
 
 # Stage2: setup runtime container
-FROM ghcr.io/cybozu/ubuntu:22.04
+FROM ghcr.io/cybozu/ubuntu:24.04
```

Comment:
@zeroalphat 
The runtime container will now use a different version of Ubuntu than that for the build.
If you have confirmed that the binaries are statically built, this would be safe.

Reply:
I fixed it so that the Ubuntu versions for build and runtime are the same.